docs/src/styles/docs/example.scss

```diff
@@ -50,9 +46,18 @@
   }
 
   & .example-copy-button {
-    background: var(--amplify-colors-background-secondary);
+    position: absolute;
     bottom: var(--amplify-space-small);
-    top: auto;
     right: var(--amplify-space-medium);
   }
 }
+
+.docs-example__preview,
+.docs-example > *:first-child {
```

Comment:
This is to not break examples that don't use the new `ExamplePreview` component to wrap the preview.